### ISSUE TITLE
Feature request: support for BigBlueButton video chat

### ISSUE THREAD

**Issue by rnsc** [RocketChat/Rocket.Chat.ReactNative#1959]
Hello,

I was wondering if it would be possible to support BigBlueButton meetings inside the iOS application.

Kind regards,

rnsc

**Comment by ReimarBauer** [comment]
Can this also be added to the android application?

**Comment by jenova15** [comment]
Same here - we use BBB server for video conferencing for Moodle and Rocket Chat desktop clients but we also have both Android and iOS Rocket Chat users who could use this feature.

**Comment by alt-alex** [comment]
Hello, I'm also interested in adding this feature to an android app.

**Comment by diegolmello** [comment]
When we do this, we're going to implement for both iOS and Android, but we don't have hands for this right now.
Maybe someone from the community could take it?

**Comment by trinh-ngo-ntq** [comment]
@diegolmello so sorry for my sudden request but do you guys have any milestones about this feature?
Rocketchat gonna be rock if calling is not available in Mobile

**Comment by diegolmello** [comment]
@trinh-ngo-ntq we do support Jitsi for calls.

**Comment by trinh-ngo-ntq** [comment]
@diegolmello Yeah i see that plugin, it works great on Mobile (rocketchat-jitsi-meet) but currently no support for BBB, right? We bang our head to stone now, no clue to start.
My project picks BBB for Video call conference because its more mature and stable, then now we must support on mobile version.

p/s: Is Ok to tag you like this, i m afraid that your mailbox gonna be full soon, please pardon me

**Comment by diegolmello** [comment]
@trinh-ngo-ntq we support videos call already and now we're working on other features, like E2E encryption, push encryption, screen lock (released this week on 4.7.0).
That's why I said above that we don't have hand for this.
We need to work on missing pieces on the app before getting back on this and adding support to another video call plugin.

I agree BBB is awesome and I'd love to review PRs adding support for it.

**Comment by terra-assistance** [comment]
@diegolmello any changes on this topic?
